apps/backoffice/openapi.yaml

```diff
@@ -0,0 +1,817 @@
+openapi: 3.0.3
+info:
+  title: IO BackOffice
+  description: |-
+    _This is the IO BackOffice Backend API specification based on the OpenAPI 3.0._
+
+    ### Some useful links:
+    - [IO-SERVICES-CMS Github codebase](https://github.com/pagopa/io-services-cms)
+    - [IO-SERVICES-CMS Source API definition ](https://github.com/pagopa/io-services-cms/openapi.yaml)
+
+  contact:
+    name: PagoPA S.p.A.
+    url: https://docs.pagopa.it/io-guida-tecnica/
+  version: 0.1.0
+servers:
+  - url: http://localhost:3000/api #FIXME
+tags:
+  - name: auth
+    description: Authentication API specification
+  - name: services
+    description: Services API specification
+  - name: service-review
+    description: Service Review API specification
+  - name: service-release
+    description: Service Release API specification
+  - name: service-authorization
+    description: Service Authorization API specification
+  - name: service-migration
+    description: Services migration API specification
+  - name: institutions
+    description: Institutions API specification
+paths:
+  /info:
+    get:
+      description: Get runtime status information about the application
+      operationId: info
+      security: [] # public endpoint
+      responses:
+        "200":
+          description: The application is up and running
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  version:
+                    type: string
+                    description: the application version
+                  name:
+                    type: string
+                    description: the application name
+        "500":
+          description: The fails to startup due to misconfiguration or unreachable dependencies
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ResponseError"
+  /auth: # /idp/selfcare/resolve-identity
+    get:
+      tags:
+        - auth
+      summary: Resolve selfcare identity
+      description: Resolve selfcare identity
+      operationId: resolveSelfCareIdentity
+      security: [] # public endpoint
+      parameters:
+        - name: identity-token
+          in: header
```

Comment:
**nitpick:** I prefer to pass it as a request body instead of a header param (to try to stay more "closely" to the oauth protocol)

Reply:
a42e8f6